flink-yarn/src/main/java/org/apache/flink/yarn/entrypoint/YarnJobClusterEntrypoint.java

```diff
@@ -131,6 +133,17 @@ protected JobGraph retrieveJobGraph(Configuration configuration) throws FlinkExc
 		}
 	}
 
+	@Override
+	protected void registerShutdownActions(CompletableFuture<ApplicationStatus> terminationFuture) {
+		terminationFuture.whenComplete((status, throwable) ->
```

Comment:
`throwable` isn't used. If `terminationFuture` cannot be completed exceptionally, `thenAccept` should be used.

Reply:
Will update